odoo/addons/base/data/res_partner_demo.xml

```diff
@@ -187,6 +187,7 @@
             <field name="function">Senior Consultant</field>
             <field name="email">jesse.brown74@example.com</field>
             <field name="phone">(829)-386-3277</field>
+            <field name="company_id" ref="base.main_company"/>
```

Comment:
@RomainLibert 
https://github.com/odoo/odoo/blob/master/odoo/addons/base/tests/test_api.py#L198
because of the test case fail, I have done these changes.